src/index.html

```diff
@@ -4,9 +4,250 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/styles.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          href="#home"
+          class="header__logo-img"
+          src="images/logo.png"
+          alt="Logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#" class="nav__list-link is-active">Apple</a>
+          </li>
+          <li>
+            <a href="#" class="nav__list-link">Samsung</a>
+          </li>
+          <li>
+            <a href="#" class="nav__list-link">Smartphones</a>
+          </li>
+          <li>
+            <a href="#laptopscomputers"
+             class="nav__list-link"
+             data-qa="nav-hover"
```

Comment:
format according to [line-wrapping](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping)